Environment-Management/SDDC_Documentation.md

```diff
@@ -0,0 +1,1026 @@
+# Operational Handbook for SDDC Administration
+
+This document contains information about how to use the SDDC Feature with Exasol. 
+
+
+## Table of Contents
+[Operational Handbook for SDDC Administration](#operational-handbook-for-sddc-administration)
+  - [Table of Contents](#table-of-contents)
+  - [Foreword](#foreword)
+  - [Background – how SDDC works](#background--how-sddc-works)
+  - [SDDC Installation](#sddc-installation)
+    - [Cluster Interconnect: Requirements and Bandwidth Planning](#cluster-interconnect-requirements-and-bandwidth-planning)
+    - [Installation Example](#installation-example)
+  - [SDDC Administration](#sddc-administration)
+    - [Starting a database](#starting-a-database)
+    - [Configuring a database](#configuring-a-database)
+    - [Creating new volumes](#creating-new-volumes)
+    - [Creating Backups](#creating-backups)
+    - [Activating or Deactivating Backup Schedules](#activating-or-deactivating-backup-schedules)
+    - [Switch Active Database to DR database](#switch-active-database-to-dr-database)
+    - [Switch back DR Database to PROD Database](#switch-back-dr-database-to-prod-database)
+  - [SDDC Monitoring](#sddc-monitoring)
+    - [Monitoring Volume States](#monitoring-volume-states)
+    - [Monitoring Database Info](#monitoring-database-info)
+    - [Monitoring Segments](#monitoring-segments)
+    - [Monitoring Redundancy Recovery](#monitoring-redundancy-recovery)
+  - [Active Node Failure Scenarios](#active-node-failure-scenarios)
+    - [Scenario: Transient Node Failure](#scenario-transient-node-failure)
+    - [Scenario: Persistent Node Failure](#scenario-persistent-node-failure)
+  - [Passive Node Failure Scenarios](#passive-node-failure-scenarios)
+  - [Disaster Scenarios](#disaster-scenarios)
+    - [SDDC Test scenario](#sddc-test-scenario)
+    - [Disaster of active data center (DC1)](#disaster-of-active-data-center-dc1)
+    - [Disaster of passive site DC2](#disaster-of-passive-site-dc2)
+    - [Network Failure between the data centers (Split Brain)](#network-failure-between-the-data-centers-split-brain)
+
+
+## Foreword
+
+The information and examples provided are developed with a customer's set up in mind. Illustrations are used heavily within this document in order to easily conceptualize how SDDC works. For example:
+
+![Standard SDDC setup](images/SDDC/SDDC_standard.png)
+
+
+This illustration uses colors to convey the status of databases, nodes, data segments, and volumes. Throughout this document, we are referencing a cluster with the following configuration:
```

Comment:
Relying exclusively on colors is not OK from an accessiblity perspective. Consider using different symbols or some other way to indicate states and roles.